### PR TITLE
Cart page

### PR DESCRIPTION
**Pull request by mikesch803**
1. cart page (functionality: remove the product, move to wishlist)
2. product page (functionality: added Link go to cart)
3.  **qty will be implemented once auth is done**

# files to check

- [ ] src/components/product-card/ProductCard.jsx
- [ ] src/context/cart-context.jsx
- [ ] src/pages/cart-page/Cart.jsx
- [ ] src/pages/product-page/Product.jsx
- [ ] src/pages/wishlist-page/Wishlist.jsx
- [ ] src/reducer/CartReducer.jsx

## [live link](https://agitated-tesla-b2afa9.netlify.app)

### CODE REVIEW

File: src/components/product-card/ProductCard.jsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+import { Link } from "react-router-dom";
+export function ProductCard() {
+  const { getSortedArr } = useContext(FilterContext);
+  const { wishlistDataHandler, wishlistData } = useContext(WishlistContext);
+  const { dispatch, state } = useContext(CartContext);
+  return (
+    <div className="product-component">
+      <ul className="product-list">
+        {getSortedArr.map((item) => {
+          const { src, title, brand, offer_price, mrp, _id, rating } = item;
```
Comment (sainath7878) [inline src/components/product-card/ProductCard.jsx:14]: Good that you've de-structured

**Review by sainath7878 — APPROVED** [review]
Clean and neat code. Things I liked is you de-structured objects and made different contexts.

File: src/components/product-card/ProductCard.jsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+                  <h3 className="card-desc-title">
+                    {title}
+                    <span className=" rating-num">
+                      {rating}
+                      <RatingIcon />
+                    </span>
+                  </h3>
+                  <p>by {brand}</p>
+                  <span className="card-price">Rs {offer_price}</span>
+                  <del className="card-mrp">Rs {mrp}</del>
```
Comment (ApurvaSawant11) [inline src/components/product-card/ProductCard.jsx:31]: Give some space between these two prices. And if possible, increase the size of `offer_price` or make it bold. That way the offer price will stand out

**Review by ApurvaSawant11 — APPROVED** [review]
Everything looks good to me!
The code is well structured and easy to understand. Just gave a UI suggestion.

**Pull request merged**